moveit_ros/planning/rdf_loader/src/rdf_loader.cpp

```diff
@@ -115,13 +126,107 @@ rdf_loader::RDFLoader::RDFLoader(TiXmlDocument *urdf_doc, TiXmlDocument *srdf_do
     srdf_.reset(new srdf::Model());
     if (!srdf_->initXml(*urdf_, srdf_doc))
     {
-      ROS_ERROR("Unable to parse SRDF");
+      ROS_ERROR_NAMED("rdf_loader", "Unable to parse SRDF");
       srdf_.reset();
     }
   }
   else
   {
-    ROS_ERROR("Unable to parse URDF");
+    ROS_ERROR_NAMED("rdf_loader", "Unable to parse URDF");
     urdf_.reset();
   }
 }
+
+bool rdf_loader::RDFLoader::isXacroFile(const std::string& path)
+{
+  // TODO: implement case-insensitive search
```

Comment:
yeah, I figured. I just thought it a good idea to point them out nevertheless. Especially as this change is about making them more accessible for users.